golem/resources/invoice/invoice.py

```diff
@@ -60,6 +60,9 @@ async def get_time_and_amount_since_latest_debit_notes(
                     )
                 break
 
+        if cumulative_time_from_all_activities == timedelta():
+            cumulative_time_from_all_activities = None  # type: ignore[assignment]
```

Comment:
Instead of silencing type linting, assign the optional type in the variable at the declaration.

Reply:
This causes more issues with mypy as "you cannot add tiemdelta to None|timedelta"